package.json

```diff
@@ -2,12 +2,14 @@
   "name": "logform",
   "version": "1.9.1",
   "description": "An mutable object-based log format designed for chaining & objectMode streams.",
-  "main": "index.js",
+  "main": "./build/index.js",
```

Comment:
This is not correct if your goal is to expose transpiled code to browser apps _(and **only** browser apps)._ (see [`resolve.mainFields`](https://webpack.js.org/configuration/resolve/#resolve-mainfields) in webpack).

The (more) correct way to do this is to make the output of the `babel` transpilation step the source value for **only** the `"browser"` field in your `package.json`. In that approach:

- node gets `"main"`
- webpack gets `"browser"` 

and everyone is happy since `node` doesn't have to debug transpiled source.

Reply:
@indexzero That makes total sense. I was not aware of the `browser` field in `package.json`, so thank you for bringing it to my attention. My original goal with the PR was to _only_ ship the transpiled code, but I definitely see the value in shipping both - especially for debugging!

Based on this new-found knowledge, there will probably need to be additional changes to my other 2 pull requests: https://github.com/winstonjs/winston-transport/pull/30 and https://github.com/winstonjs/winston/pull/1418. If you need the help I'd be happy to take another stab at the PRs, otherwise I'm fine to step back and leave it in your capable hands.

Thanks!

Reply:
@mfrisbey We’d love your help if you want to take a stab at revising your winston-transport and winston PRs (probably need one for triple-beam too) to follow the pattern used here for logform.  If you don’t have time, I’ll probably work on updating those PRs, but the more help we can get the faster all these features will land :) Thanks again!